services/src/test/scala/cromwell/pipeline/service/ProjectConfigurationServiceTestImpl.scala

```diff
@@ -0,0 +1,41 @@
+package cromwell.pipeline.service
+
+import cromwell.pipeline.datastorage.dto._
+import cromwell.pipeline.model.wrapper.UserId
+
+import scala.concurrent.Future
+
+class ProjectConfigurationServiceTestImpl(
+  projectConfiguration: ProjectConfiguration,
+  testMode: TestMode
+) extends ProjectConfigurationService {
+
+  override def addConfiguration(projectConfiguration: ProjectConfiguration, userId: UserId): Future[Unit] =
+    testMode match {
+      case WithException(exc) => Future.failed(exc)
+      case _                  => Future.unit
+    }
+
+  override def getLastByProjectId(projectId: ProjectId, userId: UserId): Future[Option[ProjectConfiguration]] =
+    testMode match {
+      case WithException(exc) => Future.failed(exc)
+      case ConstNone          => Future.successful(None)
```

Comment:
Use `projectConfigurations: Seq[ProjectConfiguration]` instead of `projectConfiguration: ProjectConfiguration` and you won't need `ConstNone` anymore

Reply:
Removed